e2e/__tests__/__snapshots__/watch_mode_only_failed.test.js.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`can press "f" to run only failed tests: test results 1`] = `
+"|                                           ^
```

Comment:
what's up with the `^`?

Reply:
seems like part of a codeframe snuck in

Reply:
In order to have a deterministic snapshot, we are sorting the lines, which brings the caret that is below `toMatchSnapshot` all the way to the top.

```
    +     Received value does not match stored snapshot "bar 1".
    +
    +     - Snapshot
    +     + Received
    +
    +     - "foo"
    +     + "bar"
    +
    +       1 |
    +     > 2 |       test('bar', () => { expect('bar').toMatchSnapshot(); });
    +         |                                         ^
    +       3 |
    +
    +       at __tests__/bar.spec.js:2:41
```

Reply:
It would be great to change the implementation of sort lines to handle this, but it sounds more complicated... I'll give it a shot anyway, because the snapshots are going to be way better